tools/ds-identify

```diff
@@ -267,6 +288,14 @@ detect_virt() {
         if [ $r -eq 0 ] || { [ $r -ne 0 ] && [ "$out" = "none" ]; }; then
             virt="$out"
         fi
+    elif [ "$DI_UNAME_KERNEL_NAME" = "FreeBSD" ]; then
+        out=$(sysctl -qn kern.vm_guest 2>/dev/null) && {
+            # Map FreeBSD's name for Hyper-V to that of systemd-detect-virt
+            if [ "$out" = "hv" ]; then
```

Comment:
maybe a comment that these values *do* match up with their detect-virt cousins.  I would never have expected such luck.

Reply:
done!
and discovered that oracle != vbox